test/test_meta.py

```diff
@@ -1179,6 +1172,20 @@ def test_fill_alias_relationship(self):
         r2 = torch.ops.aten.fill(inps, 1.0)
         self.assertNotEqual(id(inps), id(r2))
 
+    # opinfo test is using aten.fill_, it's not testing aten.fill
+    @onlyCUDA
+    def test_fill_stride(self):
```

Comment:
Shouldn't we just add an opinfo for fill?

Reply:
Yes, we should... 
adding Opinfo will probably expose more problems and should be done in another PR.
I will open an issue to track this

Reply:
https://github.com/pytorch/pytorch/issues/88112